common/src/main/java/org/ergoplatform/appkit/ErgoAuthUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.ergoplatform.appkit;
+
+import java.nio.charset.StandardCharsets;
+
+import sigmastate.Values.SigmaBoolean;
+
+/**
+ * Helper utilities for EIP-28 ErgoAuth
+ */
+public class ErgoAuthUtils {
+
+    /**
+     * Serializes the given SigmaBoolean to be sent in ErgoAuthRequest.
+     */
+    public static byte[] serializeSigmaBoolean(SigmaBoolean sigmaBoolean) {
+        return Iso.isoSigmaBooleanToByteArray().to(sigmaBoolean);
+    }
+
+    /**
+     * Converts given address to SigmaBoolean and serializes it  to be sent in ErgoAuthRequest.
+     */
+    public static byte[] serializeSigmaBoolean(Address address) {
```

Comment:
This method creates confusion. It looks like overload of the above method, implying that Address and SigmaBoolean are the same (or isomorphic), but they are far from that.
It is better to remove this method and instead add Address.getSigmaBoolean method. 
Also Address.isSigmaBoolean is necessary, because not every Address is backed by SigmaBoolean. 
Note p2pk is a specific case of SigmaBoolean address.

Reply:
Changed